sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -315,6 +315,49 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.select(isnotnull(col("a"))), Seq(Row(false)))
   }
 
+  test("Test nullif function") {
```

Comment:
Let's follow the convention in the file, and name the test as:
```suggestion
  test("nullif function") {
```